.github/workflows/3x_nightly_tests.yml

```diff
@@ -0,0 +1,100 @@
+on:
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  # Checkout in separate job because docker image is alpine based and checkout action doesn't work.
+  checkout_build:
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/pantheon-public/php-ci:v7.4
+    name: Checkout & build Phar
+    env:
+      TZ: "/usr/share/zoneinfo/America/Los_Angeles"
+      TERM: dumb
+    steps:
+      - name: Checkout 3.x branch
+        uses: actions/checkout@v2
+        with:
+          ref: 'refs/heads/3.x'
+      - name: Save repo content as artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: full-workspace
+          path: /home/runner/work/terminus/terminus
+      - name: Full Composer Install
+        run: composer install
+      - name: Validate Code
+        run: composer code:lint
+      - name: Phar Build
+        run: composer phar:build
+      - name: permissions
+        run: chmod +x ./t3
+      - name: Save t3.phar as artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: t3-phar
+          path: t3
+          if-no-files-found: error
+
+  functional:
+    runs-on: ${{ matrix.operating-system }}
+    name: Functional testing matrix - PHP ${{ matrix.php-versions }} on ${{ matrix.operating-system }}
+    if: ${{ ! startsWith(github.ref, 'refs/tags/')  && github.repository_owner == 'pantheon-systems' }}
+    strategy:
+      matrix:
+        operating-system: [ 'macos-latest' ]
+        php-versions: [ '8.0' ]
+      max-parallel: 1
+    env:
+      TZ: "/usr/share/zoneinfo/America/Los_Angeles"
+      TERM: dumb
+      COMPOSER_PROCESS_TIMEOUT: 0
+      COMPOSER_ALLOW_SUPERUSER: 1
+      TERMINUS_TOKEN: ${{ secrets.TERMINUS_TOKEN }}
+      TERMINUS_SITE: ${{ secrets.TERMINUS_SITE }}
+    needs: [ checkout_build ]
+    steps:
+      - name: Install SSH key
+        uses: webfactory/ssh-agent@v0.5.3
+        with:
+          ssh-private-key: ${{ secrets.TERMINUS_SITE_OWNER_SSH_PRIVATE_KEY }}
+      - name: Setup PHP with PECL extension
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+          extensions: gd, mbstring, zip, ssh2-1.3.1
+          coverage: pcov
+      - name: Download repo content from artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: full-workspace
+      - name: Download terminus.phar as artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: t3-phar
+      - name: Install Composer Dependencies
+        run: composer install --no-interaction --prefer-dist
+      - name: Functional Tests (short)
+        run: composer test:short
+        env:
+          COMPOSER_ALLOW_SUPERUSER: 1
+          TERMINUS_TOKEN: ${{ secrets.TERMINUS_TOKEN }}
+          TERMINUS_SITE: ${{ secrets.TERMINUS_SITE }}
+          TERMINUS_ORG: ${{ secrets.TERMINUS_ORG }}
+      - name: Functional Tests (long)
+        run: composer test:long
+        env:
```

Comment:
See comment on line 82

Reply:
@namespacebrian (commented above). This was copied from the original `ci.yml`, will optimize this later.

Reply:
Yeah, I'd noticed it before in our existing functional test config.  I realize it's just being relocated here with no changes.  It's a minor thing.